suite-native/module-settings/src/components/ApplicationSettings.tsx

```diff
@@ -9,12 +11,30 @@ import {
     RootStackParamList,
     RootStackRoutes,
 } from '@suite-native/navigation';
+import { Text } from '@suite-native/atoms';
 import { isDevelopOrDebugEnv } from '@suite-native/config';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
 import { SettingsSection } from './SettingsSection';
 import { SettingsSectionItem } from './SettingsSectionItem';
 
+const devButtonStyle = prepareNativeStyle(() => ({
+    position: 'absolute',
+    zIndex: 1,
+    top: 0,
+    right: 0,
+    width: 50,
+    height: 50,
+}));
+
+const isDevButtonVisibleAtom = atom<boolean>(false);
```

Comment:
Change init value to:
```suggestion
const isDevButtonVisibleAtom = atom<boolean>(isDevelopOrDebugEnv());
```

Then you can remove this line:
   ```ts
 const shouldShowDevButton = isDevelopOrDebugEnv() || isVisible;
```

Reply:
https://github.com/trezor/trezor-suite/pull/7962/commits/564357fd962741e9197a6de461ee8b7fb9d41960